lib/features/sks-menu/presentation/widgets/technical_message.dart

```diff
@@ -0,0 +1,41 @@
+import "package:flutter/material.dart";
+
+import "../../../../config/ui_config.dart";
+import "../../../../theme/app_theme.dart";
+import "../../data/models/dish_category_enum.dart";
+
+class TechnicalMessage extends StatelessWidget {
+  const TechnicalMessage({
+    super.key,
+    required this.message,
+    this.title,
+    this.color,
+  });
+  final String message;
+  final String? title;
+  final Color? color;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(
+        HomeViewConfig.paddingMedium,
+      ),
+      child: ClipRRect(
+        borderRadius: BorderRadius.circular(8),
```

Comment:
[nitpick alert] maybe we should move it to config?

Reply:
I'll probably reuse something we already have in the config